priv/repo/seeds.exs

```diff
@@ -251,7 +252,7 @@ currency_rewards =
 
 currency_rewards =
   currency_rewards ++
-    Enum.map(levels_without_units, fn level ->
+    Enum.map(levels_without_embeds, fn level ->
```

Comment:
Let's fix this double iteration while we're at it! We can use a [flat_map](https://hexdocs.pm/elixir/1.12/Enum.html#flat_map/2)